integration-tests/http/__tests__/returns/returns.spec.ts

```diff
@@ -233,6 +233,22 @@ medusaIntegrationTestRunner({
           })
         )
 
+        // remove shipping method
+        const action_id = result.data.order.shipping_methods[1].actions[0].id
```

Comment:
question: Should we add an assertion in between the deletion and creation to ensure it is deleted correctly?

Reply:
absolutely, I got distracted fixing wierd bugs when this was added and forgot about getting back to it 😆